.github/workflows/ci-cpp-build-gnoi.yml

```diff
@@ -0,0 +1,37 @@
+name: "build gnoi"
```

Comment:
how about "bazel build"? gnoi should be obvious from context.

Reply:
Done